### DISCUSSION TITLE
[Closed] Moving to the new .NET Foundation CLA service

### DISCUSSION THREAD

**Discussion by dnfadmin** [CommunityToolkit/Maui#1044] [Announcements]
Hello everyone
The .NET Foundation is transitioning to a new CLA service and I'm posting here to let you know it's happening. Some background info:

The new CLA service is based off of https://github.com/microsoft/ContributorLicenseAgreement, except it uses the dotnet-policy-service app

- Current contributors do not need to sign the CLA again
- The CLA agreement itself is not changing
- As part of the install/configuration process, the following work items will be completed

1. Create a .github repo if it doesn't exist already
2. Install the dotnet-policy-service app
3. Create PRs for the platformcontext.xml and cla.yml files
4. Disable the existing CLA webhook
5. Push the PRs to enable the new CLA service

That's all there is to it. I'll update this issue once the work is complete

Thanks
Chris
.NET Foundation Operations team

**Comment by TheCodeTraveler** [comment]
Thanks @dnfadmin! Happy to help.

I have a couple questions before I can implement this:
- Where do I find the `dotnet-policy-service` app, and how do I "install" it?
- Where do I find the `platformcontext.xml` and `cla.yml` files? 
  - And where do I add these files in our repository
- How do I disable the existing CLA webhook?

**Comment by dnfadmin** [comment]
Hi @brminnick - we took care of all this work for you :-)  The new CLA is now operational

**Reply by TheCodeTraveler** [reply]
Great! Thank you!

**Comment by rmarinho** [comment]
@dnfadmin can you check this on the dotnet/maui repo? seems we are having some issues

**Comment by CliffAgius** [comment]
Closed as completed.
